lib/player.js

```diff
@@ -2277,8 +2279,15 @@ shaka.Player = class extends shaka.util.FakeEventTarget {
       this.onAbrStatusChanged_();
     }
 
-    // Re-filter the manifest after streams have been chosen.
-    this.filterManifestByCurrentVariant_();
+    // // Re-filter the manifest after streams have been chosen.
+    // // Note: Skip if an audio codec switching strategy is being enforced.
+    // if (this.config_.mediaSource.codecSwitchingStrategy ===
```

Comment:
Is this meant to be part of a follow-up commit or something? Just having some random commented-out code lying around is weird.

Reply:
Removed!